addons/pos_self_order/static/src/app/pages/stand_number_page/stand_number_page.js

```diff
@@ -35,6 +35,7 @@ export class StandNumberPage extends Component {
     confirm() {
         if (this.state.standNumber.length > 0) {
             this.selfOrder.tablePadNumber = this.state.standNumber;
+            this.selfOrder.currentOrder.table_stand_number = this.state.standNumber;
```

Comment:
Nice I see that table_stand_number is now saved on the order. You should adapt it everywhere in the code now.

For example, `payment_success_page.xml` awalys use the old behavior:

```
<span class="number m-0" t-esc="selfOrder.tablePadNumber" />
```